src/lib/puppeteer/screenshot/getInstagramPostReelScreenshot.ts

```diff
@@ -97,6 +98,9 @@ async function navigateCarousel(
 	const { index, logger, page } = options;
 
 	// Handle dialogs only if index is greater than 1 so that we can get the thumbnail image of the video before it starts
+	//this is a separate function to handle dialogs only for instagram
+	await handleDialogsInstagram({ logger, page });
+	//this function act as a fallback if the handleDialogsInstagram fails
```

Comment:
```suggestion
	// This is a separate function to handle dialogs only for instagram
	await handleInstagramDialogs({ logger, page });
	// This function act as a fallback if the handleInstagramDialogs fails
```

Please follow how other parts of the codebase is coded, we have the comments starting with capital and a space left after `//`.